changelog/6332.feature.rst

```diff
@@ -0,0 +1 @@
+Added the plot_equator_prime_meridian.py example to the Example Gallery.
```

Comment:
Ideally we want to link to the gallery example if you can look at: https://sphinx-gallery.github.io/stable/advanced.html#cross-referencing